### PR TITLE
Licenses fix

### PR DESCRIPTION
**Pull request by biofotis**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropiate.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is a reminder of what we should look for before merging this code. I have:_ 

- [ ] Read and follow the [contributing guidelines](https://github.com/shadow-robot/sr_documentation/blob/master/CONTRIBUTING.md).
- [ ] Checked that all tests pass with my changes
- [ ] Added tests (automatic or manual) that prove the fix is effective or that the feature works
- [ ] Added necessary documentation (if appropriate)
- [x] Added the corresponding license to each file and add the current year of any one that you modified.
- [ ] Tested on real hardware (if appropriate)

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: sr_hand/include/sr_hand/hand/CAN_compatibility_arm.hpp [hunk 1]
```diff
@@ -1,24 +1,25 @@
-/**
- * @file   CAN_compatibility_arm.hpp
- * @author Ugo Cupcic <ugo@shadowrobot.com>
- * @author Guillaume Walck (UPMC)
- * @date   Sun Nov 20 2011
- *
-* Copyright 2011 Shadow Robot Company Ltd.
+/*
+* Copyright 2010 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_hand/include/sr_hand/hand/CAN_compatibility_arm.hpp]:
```suggestion
* Copyright 2011 Shadow Robot Company Ltd.
```

File: sr_hand/include/sr_hand/hand/etherCAT_compatibility_hand.hpp [hunk 2]
```diff
@@ -1,23 +1,24 @@
-/**
- * @file   etherCAT_compatibility_hand.hpp
- * @author Ugo Cupcic <ugo@shadowrobot.com>
- * @date   Mon Aug 22 10:33:35 2011
- *
-* Copyright 2011 Shadow Robot Company Ltd.
+/*
+* Copyright 2010 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_hand/include/sr_hand/hand/etherCAT_compatibility_hand.hpp]:
```suggestion
* Copyright 2011 Shadow Robot Company Ltd.
```

File: sr_hand/include/sr_hand/hand_commander.hpp [hunk 3]
```diff
@@ -1,24 +1,24 @@
-/**
- * @file   hand_commander.hpp
- * @author Toni Oliver <toni@shadowrobot.com>, Contact <contact@shadowrobot.com>
- * @date   Thu Nov 08 15:34:37 2012
- *
-*
-* Copyright 2011 Shadow Robot Company Ltd.
+/*
+* Copyright 2010 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_hand/include/sr_hand/hand_commander.hpp]:
```suggestion
* Copyright 2012 Shadow Robot Company Ltd.
```

File: sr_hand/scripts/sr_hand/__init__.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2011 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_hand/scripts/sr_hand/__init__.py]: I think this is wrong. This file is usually empty

File: sr_hand/src/hand/real_arm.cpp [hunk 5]
```diff
@@ -1,24 +1,24 @@
-/**
- * @file   real_arm.cpp
- * @author Ugo Cupcic <ugo@shadowrobot.com>, Contact <contact@shadowrobot.com>
- * @date   Tue Jun 29 14:56:10 2010
- *
-*
+/*
 * Copyright 2011 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_hand/src/hand/real_arm.cpp]:
```suggestion
* Copyright 2010 Shadow Robot Company Ltd.
```

File: sr_hand/src/hand_commander.cpp [hunk 6]
```diff
@@ -1,24 +1,24 @@
-/**
- * @file   hand_commander.cpp
- * @author Toni Oliver <toni@shadowrobot.com>, Contact <contact@shadowrobot.com>
- * @date   Thu Nov 08 15:34:37 2012
- *
-*
-* Copyright 2011 Shadow Robot Company Ltd.
+/*
+* Copyright 2010 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_hand/src/hand_commander.cpp]:
```suggestion
* Copyright 2012 Shadow Robot Company Ltd.
```

File: sr_mechanism_model/src/null_transmission.cpp [hunk 7]
```diff
@@ -1,36 +1,19 @@
...
- *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
- *  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
- *  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
- *  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
- *  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
- *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
- *  POSSIBILITY OF SUCH DAMAGE.
- *********************************************************************/
+/*
+* Copyright 2011 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_mechanism_model/src/null_transmission.cpp]:
```suggestion
* Copyright 2008 Shadow Robot Company Ltd.
```

File: sr_tactile_sensors/scripts/sr_tactile_sensors/__init__.py [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2011 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_tactile_sensors/scripts/sr_tactile_sensors/__init__.py]: Again no needed

File: sr_utilities/scripts/__init__.py [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2011 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_utilities/scripts/__init__.py]: Not needed

File: sr_utilities/scripts/sr_utilities/__init__.py [hunk 10]
```diff
@@ -1 +1,15 @@
+# Copyright 2011 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_utilities/scripts/sr_utilities/__init__.py]: Not needed

File: sr_utilities/scripts/sr_utilities/grasp_controller_spawner.py [hunk 11]
```diff
@@ -1,20 +1,19 @@
 #!/usr/bin/env python
-#
-# Copyright 2015 Shadow Robot Company Ltd.
+
+# Copyright 2014 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_utilities/scripts/sr_utilities/grasp_controller_spawner.py]:
```suggestion
# Copyright 2015 Shadow Robot Company Ltd.
```

File: sr_utilities/test/test_arm_finder.cpp [hunk 12]
```diff
@@ -1,3 +1,19 @@
+/*
+* Copyright 2015 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_utilities/test/test_arm_finder.cpp:2]: If we don't know the year of the original, maybe 2020 would be better?

File: sr_utilities/test/test_arm_finder.py [hunk 13]
```diff
@@ -1,5 +1,19 @@
 #!/usr/bin/env python
 
+# Copyright 2011 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_utilities/test/test_arm_finder.py:3]: If we don't know the year of the original, maybe 2020 would be better?

File: sr_utilities/test/test_hand_finder.cpp [hunk 14]
```diff
@@ -1,3 +1,19 @@
+/*
+* Copyright 2011 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_utilities/test/test_hand_finder.cpp:2]: If we don't know the year of the original, maybe 2020 would be better?

File: sr_utilities/test/test_hand_finder.py [hunk 15]
```diff
@@ -1,5 +1,19 @@
 #!/usr/bin/env python
 
+# Copyright 2011 Shadow Robot Company Ltd.
```
Comment (beatrizleon) [inline sr_utilities/test/test_hand_finder.py:3]: If we don't know the year of the original, maybe 2020 would be better?

**Review by beatrizleon — CHANGES_REQUESTED** [review]

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**